### PR TITLE
DOCS-4466-Added bandwidth requirement

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/install-port-reqs.adoc [hunk 1]
```diff
@@ -5,6 +5,13 @@ endif::[]
 
 Runtime Fabric requires specific network and port configuration for installation and operation.
 
+== Bandwidth Requirements
+
+A high bandwidth network is required, with a minimum of 1 GBps connectivity in the internal network running the RTF nodes.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-port-reqs.adoc]: consider rewording -- suggestion: "..between each node operating Runtime Fabric."

File: modules/ROOT/pages/install-port-reqs.adoc [hunk 2]
```diff
@@ -5,6 +5,13 @@ endif::[]
 
 Runtime Fabric requires specific network and port configuration for installation and operation.
 
+== Bandwidth Requirements
+
+A high bandwidth network is required, with a minimum of 1 GBps connectivity in the internal network running the RTF nodes. 
+[NOTE]
+Due to data center connectivity being lower than the minimum requirement, cross-regional deployments of a Runtime Fabric
```
Comment (sundippatel) [inline modules/ROOT/pages/install-port-reqs.adoc]: We can be blunt here: "Cross-regional deployments of a single Runtime Fabric is not supported."
If we list reasons, customers may claim they have a set up that doesn't apply.

File: modules/ROOT/pages/install-port-reqs.adoc (hunk 1 above)
Comment (sundippatel) [reply modules/ROOT/pages/install-port-reqs.adoc]: We should also add a requirement with outbound connectivity to the Internet: 100 mbps.

**Review by sundippatel — CHANGES_REQUESTED** [review]
Left some comments. Thanks for adding in this bandwidth requirement. Long overdue!

File: modules/ROOT/pages/install-port-reqs.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-port-reqs.adoc]: Got it - thank you!

File: modules/ROOT/pages/install-port-reqs.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-port-reqs.adoc]: I removed that part - thank you!

**Review by sarathecoubian — APPROVED** [review]
Looks good to me!

**Review by sundippatel — APPROVED** [review]

**Pull request merged**